kolibri/core/auth/api.py

```diff
@@ -74,6 +79,33 @@
 from kolibri.plugins.app.utils import interface
 
 
+class BasicAuthenticationFacility(BasicAuthentication):
```

Comment:
Morango [allows something similar](https://github.com/learningequality/morango/blob/master/morango/api/permissions.py#L13) which I also needed to take advantage of by passing the facility ID.

Reply:
Oh, I was unaware of the morango implementation when @jredrejo and I discussed this. Would it be worth just reusing that?

Reply:
The Morango class is a little more generic/flexible which doesn't necessarily make it super clear that we're passing through additional parameters, but I think that also makes it less likely to change and little more future-proof, so reusing it probably wouldn't hurt!

Reply:
I think the facility ID is needed because there might be these scenarios in one device:
one user has been copied from one facility to another, having seme credentials in both facilities
or by casuality someone  has chosen the same username and password in two different facilities 

How do you know which user is trying to use the authentication?

Reply:
@jredrejo You're correct, it needs the facility ID for authentication. The point that I was making is that Morango has a class that allows for passing arbitrary additional parameters, like the facility ID, just less explicitly than you have coded here. So it's possible we could simply use that class instead.

Reply:
@bjester  @rtibbles  done, but beware that I've had to fix some issues that were important in kolibri and have raised when using this authentication. They are at #8298 for a separate aproval, but cherry-picked here or authentication didn't work

Reply:
Thanks again @bjester  for your help, everything is working now using `BasicMultiArgumentAuthentication` from Morango